src/components/v2/V2Create/ConfirmDeployV2ProjectModal.tsx

```diff
@@ -0,0 +1,133 @@
+import { t, Trans } from '@lingui/macro'
```

Comment:
This component is scrapy at the moment, but serves the purpose of the MVP. We'll flesh it out with more data/polish over time.